### ISSUE TITLE
dedupingInterval not working on vue 3?

### ISSUE THREAD

**Issue by ConsoleTVs** [Kong/swrv#76]
I have setup `dedupingInterval` but it's not working, it requests it every time I change to that page, even if inside the safe internval. It should not make a request.

**Comment by ConsoleTVs** [comment]
Note: using: `"swrv": "^1.0.0-beta.1"`

**Comment by darrenjennings** [comment]
not sure it ever worked. I think it really only dedupes based on in-flight promises. Keeping this open as a bug because we need both configs I think:

dedupingInterval: 2000
```
----------1000--------------2000--------------3000--------------4000 ms
r1('c1') ----------->|
            r2('c1')
            r3('c2') ---------------------------------------->|
                                                     r4('c2')-------->|
```
r2 is deduped
r4 is not deduped (because deduping interval)

We should make it to where both r2 and r4 would be deduped.

**Comment by ConsoleTVs** [comment]
Sadly, I ended up creating this library myself with the addition of other features I found missing here. I added a small FAQ in with the reason / comparison of both libs. I needed this for work and I could not wait for this to get fixed. Nevertheless, I don't see this as competition but rather good news. Feel free to check out my code in case you're interested in the solution. I basically used cache expiration times to resolve if I had to fetch or not and therefore, implementing deduping pretty easy.

Let me know any concerns. I won't close the issue since this is an active bug as pointed by @darrenjennings 

The alternative I created is: https://github.com/ConsoleTVs/vswr
